python/ray/autoscaler/_private/aws/cloudwatch/cloudwatch_helper.py

```diff
@@ -420,6 +447,23 @@ def _replace_dashboard_config_variables(self) -> Dict[str, Any]:
             widgets.append(item_out)
         return widgets
 
+    def _replace_alarm_config_variables(self) -> Dict[str, Any]:
+        """
+        replace known variable occurrences in cloudwatch alarm config file
```

Comment:
Can we enumerate what the known variables are in this docstring (instance ID, cluster name, and region)?

Reply:
Sure, added variables description in _replace_cwa_config_variables(), _replace_dashboard_config_variables() and _replace_alarm_config_variables() functions.